README.md

```diff
@@ -1 +1,45 @@
-# samples-rust
\ No newline at end of file
+<h1 align="center"> Keploy x Rust </h1>
+<p align="center">
+
+  <a href="CODE_OF_CONDUCT.md" alt="Contributions welcome">
+    <img src="https://img.shields.io/badge/Keploy-Rust-orange" /></a>
+
+  <a href="CODE_OF_CONDUCT.md" alt="Contributions welcome">
+    <img src="https://img.shields.io/badge/Contributions-Welcome-brightgreen?logo=github" /></a>
+
+  <a href="https://join.slack.com/t/keploy/shared_invite/zt-12rfbvc01-o54cOG0X1G6eVJTuI_orSA" alt="Slack">
+    <img src=".github/slack.svg" /></a>
+
+  <a href="https://opensource.org/licenses/Apache-2.0" alt="License">
+    <img src=".github/License-Apache_2.0-blue.svg" /></a>
+
+  <a href="https://opensource.org/licenses/Apache-2.0" alt="License">
+    <img src=".github/License-Apache_2.0-blue.svg" /></a>
```

Comment:
licence is present twice! And images are still breaking please check once!

Reply:
![image](https://github.com/keploy/samples-rust/assets/63868528/afd3dc2a-bda2-486a-8ac8-907d59f9eba5)
The images aren't breaking locally. Fixed the license part!!